modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -54,243 +47,294 @@ namespace {
-
-        // [[codegen::verbatim(ColorQuantityInfo.description)]]
-        std::optional<int> colorQuantity;
-
-        // List of ranges for which their corresponding parameters values will be
-        // colorized by. Should be entered as {min value, max value} per range
+        // Ranges for which their corresponding parameters values will be
+        // colorized by. Should be entered as min value, max value
         std::optional<std::vector<glm::vec2>> colorTableRanges;
+        // Specifies the total range
+        std::optional<glm::vec2> colorMinMaxRange;
 
-        // Enables flow, showing the direction, but not accurate speed, that particles
-        // would be traveling
+        // [[codegen::verbatim(FlowEnabledInfo.description)]]
         std::optional<bool> flowEnabled;
-
         // [[codegen::verbatim(FlowColorInfo.description)]]
         std::optional<glm::vec4> flowColor [[codegen::color()]];
-
         // [[codegen::verbatim(FlowReversedInfo.description)]]
         std::optional<bool> reversedFlow;
-
         // [[codegen::verbatim(FlowParticleSizeInfo.description)]]
         std::optional<int> particleSize;
-
         // [[codegen::verbatim(FlowParticleSpacingInfo.description)]]
         std::optional<int> particleSpacing;
-
         // [[codegen::verbatim(FlowSpeedInfo.description)]]
         std::optional<int> flowSpeed;
 
         // [[codegen::verbatim(MaskingEnabledInfo.description)]]
         std::optional<bool> maskingEnabled;
-
         // [[codegen::verbatim(MaskingQuantityInfo.description)]]
         std::optional<int> maskingQuantity;
-
-        // List of ranges for which their corresponding parameters values will be
-        // masked by. Should be entered as {min value, max value} per range
+        // Ranges for which their corresponding quantity parameter value will be
+        // masked by. Should be entered as {min value, max value}
         std::optional<std::vector<glm::vec2>> maskingRanges;
+        // Ranges for which their corresponding parameters values will be
+        // masked by. Should be entered as min value, max value
+        std::optional<glm::vec2> maskingMinMaxRange;
 
         // [[codegen::verbatim(DomainEnabledInfo.description)]]
         std::optional<bool> domainEnabled;
 
-        // Value should be path to folder where states are saved. Specifying this
-        // makes it use file type converter
-        // (JSON/CDF input => osfls output & oslfs input => JSON output)
-        std::optional<std::string> outputFolder;
-
         // [[codegen::verbatim(LineWidthInfo.description)]]
         std::optional<float> lineWidth;
 
+        // [[codegen::verbatim(ColorUseABlendingInfo.description)]]
+        std::optional<bool> alphaBlendingEnabled;
+
+        // Set if first/last file should render forever
+        bool showAtAllTimes;
+
+        // If using dynamic downloading, files are not cached by default.
+        // To save files locally , set this to true.
```

Comment:
```suggestion
        // Set this to true to save the downloaded files locally.
```
Add some info about where they are begin saved?